src/Templates/NServiceBusWindowsService/NServiceBusWindowsService.csproj

```diff
@@ -4,6 +4,8 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>net462</TargetFramework>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <!-- TODO: consider using a later version of C# -->
```

Comment:
Hmm, not sure how I feel about this, vs just using the default behavior.

Reply:
Given that the template is using SDK-style csproj, which requires VS 2017, and C# 7 is lowest possible default in VS 2017, this is redundant, so I'll remove it.